cosmos/airflow/graph.py

```diff
@@ -455,6 +456,37 @@ def calculate_detached_node_name(node: DbtNode) -> str:
     return node_name
 
 
+def _add_teardown_task(
+    dag: DAG,
+    execution_mode: ExecutionMode,
+    task_args: dict[str, Any],
+    tasks_map: dict[str, Any],
+    task_group: TaskGroup | None,
+    render_config: RenderConfig | None = None,
+) -> None:
+    if execution_mode != ExecutionMode.AIRFLOW_ASYNC:
+        return
+
+    if render_config is not None:  # TestBehavior.AFTER_ALL
+        task_args["select"] = render_config.select
+        task_args["selector"] = render_config.selector
+        task_args["exclude"] = render_config.exclude
```

Comment:
Why do we need this information for tear down?

Reply:
This helps delete only the files that need to be removed:

1. Create the command.
2. Execute the command with a mock.
3. Step (2) will give the files that should be deleted.

Reply:
Happy to remove it if we are ok to delete /run the folder directly